### PR TITLE
LCMS Hotfix

### PR DESCRIPTION
**Pull request by icaoberg**
This fix addresses the regex issue posted by @mccalluc 

This fix for the `raw_data` folder regex should now match the example provided by the TMC

```
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep01_techrep01.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep01_techrep02.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep02_techrep01.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep02_techrep02.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep03_techrep01.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep03_techrep02.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep04_techrep01.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep04_techrep02.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep05_techrep01.raw
raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep05_techrep02.raw
```

### CODE REVIEW

**Review by jswelling — APPROVED** [review]

**Comment by icaoberg** [comment]
@mccalluc the checks keep failing but I cannot recreate these errors locally and are related to running running the examples, not the changes I have made to the schema or documentation. Please advice how to proceed.

**Comment by icaoberg** [comment]
@mccalluc I looked at your GitHub Action
* I am running Ubuntu 20.04 while the Action builds on 18.04
* I am running Python 3.7 while the Action runs Python 3.6

From a comment in your action it needs to be this combination of Ubuntu+Python but I cannot recreate this error locally.

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 1]
```diff
@@ -1,16 +1,17 @@
 -
-  pattern: 'raw_data/*\.(raw|mzML)'
+  example: 'raw_data/20200707_rmi049_75umPLRPS_Kidney_GF10pc_VAN0003LK32_biorep05_techrep02.raw'
+  pattern: 'raw_data/[^/]*.(raw|mzML)'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:3]:
```suggestion
  pattern: 'raw_data/[^/]+\.(raw|mzML)'
```

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
   description: 'Raw mass spectrometry data from an assay of LC-MS, MS, LC-MS Bottom-Up, MS Bottom-Up, LC-MS Top-Down, or MS Top-Down that describes an analyte class of protein, metabolites, lipids, peptides, phosphopeptides, or glycans.'
   required: True
 -
-  pattern: 'ID_search_results/*\.(txt|csv)'
+  pattern: 'ID_search_results/[^/]*.(txt|csv)'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:7]:
```suggestion
  pattern: 'ID_search_results/[^/]+\.(txt|csv)'
```

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
   description: 'Identification results. Annotated data describing (qualitative or quantitative) the proteins, metabolites, lipids, peptides, phosphopeptides, or glycans identified from the corresponding raw data.'
   required: True
 -
-  pattern: 'ID_metadata/*\.xml'
+  pattern: 'ID_metadata/[^/]*.xml'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:11]:
```suggestion
  pattern: 'ID_metadata/[^/]+\.xml'
```

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
   description: 'Identification search parameters / metadata. Software settings used during the analyte identification process (e.g., from MaxQuant or Proteome Discoverer).'
   required: False
 -
-  pattern: 'QC_results/*\.(xml|txt|html|pdf|log|yaml)'
+  pattern: 'QC_results/[^/]*.(xml|txt|html|pdf|log|yaml)'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:15]:
```suggestion
  pattern: 'QC_results/[^/]+\.(xml|txt|html|pdf|log|yaml)'
```

**Review by mccalluc — CHANGES_REQUESTED** [review]
`.` in regex means any character. If you want a literal period, please escape.

File: src/ingest_validation_tools/directory-schemas/lcms.yaml (hunk 4 above)
Comment (icaoberg) [reply src/ingest_validation_tools/directory-schemas/lcms.yaml]: @mccalluc thanks for the suggestion.

**Review by jswelling — APPROVED** [review]

**Pull request merged**